community/dbms/src/main/java/org/neo4j/server/configuration/ConfigLoader.java

```diff
@@ -61,6 +61,12 @@ public Config loadConfig( Optional<File> homeDir, Optional<File> configFile,
                 settingsClassesSupplier );
     }
 
+    public Config loadOfflineConfig( Optional<File> homeDir, Optional<File> configFile )
+    {
+        return loadOfflineConfig( homeDir, configFile,
+                Pair.of( GraphDatabaseSettings.auth_enabled.name(), Settings.FALSE ) );
+    }
+
```

Comment:
It is called by all of our tools, and in tests.

My reasoning is, it's an offline config. E.g., all connectors are disabled. Having auth disabled is fine because there is nothing to connect to, and hence nothing to break into.

Furthermore, it is actually an override of an override, see 

https://github.com/spacecowboy/neo4j/blob/e4e651d401a759136f1e9bfb1e32870c59674566/community/dbms/src/main/java/org/neo4j/server/configuration/ConfigLoader.java#L113

which I found to be very unexpected.